public/index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Notification</title>
+        <style>
+            .notifs-container {
+                margin: auto;
+                width: 50%;
+                border: 3px solid black;
+                border-radius: 10px
+            }
+            #notif-title {
+                background-color: #eee;
+                text-align: center;
+                font-size: 25px;
+            }
+        </style>
+    </head>
+<body>
+    <div class='notifs-container'>
+        <div id='notif-title' >
+            <span>NOTIFICATIONS</span>
+        </div>
+        <div class="notifications"></div>
+    </div>
+    <script src="/socket.io/socket.io.js"></script>
+    <script>
+        const socket = io('http://localhost:8000');
```

Comment:
hey Jim could we use the base_URL env variable cause on Heroku that link won't work